### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by luckeyodeh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "material-ui": "^0.20.0",
```
[the remark this answers is not in the document]
Comment (luckeyodeh) [reply package.json]: ok

**Comment by jihdeh** [comment]
Not sure if you know how to resolve merge conflicts.
But merging to master you have resolve the conflicts here, you have to update your local branch as well.
Here's a couple of steps/commands you can follow when working off branches and with a team.
```
1. git checkout -b new-branch (create and checkout new branch)
——do stuff—commit—and more stuff——commit
2. git checkout master
3. git pull —rebase(pull updated work from master)
4. git checkout new branch
5. git rebase master(add master changes to your current new branch)
6. git push origin new_branch
```

File: src/containers/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              <li><a href="#">Music</a></li>
+              <li><a href="#">Videos</a></li>
+              <li><a href="#">Most Liked</a></li>
+              <li><a href="#">Most Viewed</a></li>
+              <li><a href="#">Winners</a></li>
+            </ul>
+        </nav>
+        </div>
+            <div className="right-b">
+              <i class="far fa-envelope right-b__icon"></i>
```
Comment (jihdeh) [inline src/containers/dashboard.js]: should be className as well as the others below,

File: src/styles/webheader.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.web-header {
+  background-color: #24243e;
+}
+
+nav {
```
Comment (jihdeh) [inline src/styles/webheader.css]: what if i'm using `<nav>` somewhere else and this style overrides the default style i want. This is why you shouldn't write class rules for elements like this unless you're sure, they class rule will be consistent everywhere.

**Comment by jihdeh** [comment]
[Resolving conflicts](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/)
Make sure to check which changes you're adding or removing

File: src/styles/webheader.css (hunk 3 above)
Comment (luckeyodeh) [reply src/styles/webheader.css]: ok

File: src/containers/dashboard.js (hunk 2 above)
Comment (luckeyodeh) [reply src/containers/dashboard.js]: ok

**Pull request merged**